lib/publishing.ts

```diff
@@ -103,7 +110,8 @@ export class PublishToNpmProject extends cdk.Construct implements IPublisher {
       entrypoint: 'publish.sh',
       environment: {
         FOR_REAL: forReal,
-        NPM_TOKEN_SECRET: props.npmTokenSecret.secretArn
+        NPM_TOKEN_SECRET: props.npmTokenSecret.secretArn,
+        DISTTAG: props.distTag || ''
```

Comment:
What's wrong in leaving `undefined` here?

Reply:
The way `environment` is defined right now, the value cannot be undefined.